src/utils/api/monitoringPlansApi.js

```diff
@@ -104,18 +104,20 @@ export const getMonitoringAnalyzerRanges = async (locId, componentRecordId) => {
 
 export const postCheckoutMonitoringPlanConfiguration = async (id, user) => {
   const userName = { username: user };
-  return axios
-    .post(
-      `${config.services.monitorPlans.uri}/workspace/plans/${id}/check-outs`,
-      userName
-    )
+  return secureAxios({
+    method: "POST",
+    url: `${config.services.monitorPlans.uri}/workspace/plans/${id}/check-outs`,
+    data: userName,
+  })
     .then((response) => response.data)
     .catch(handleError);
 };
 
 export const revertOfficialRecord = async (id) => {
-  return axios
-    .delete(`${config.services.monitorPlans.uri}/workspace/plans/${id}/revert`)
+  return secureAxios({
```

Comment:
eventually, when we switch over to secureAxios for all transactions, we can do this at the very top of the file with the imports to avoid changing axios to "secureAxios" everywhere.

import { secureAxios as axios } from "some/location";